integr8ly/workload-web-app
# MGDAPI-4171 adopt for sandbox

## JIRA
Closes [MGDAPI-4171](https://issues.redhat.com/browse/MGDAPI-4171)

## What
Adoption of the workload web-app to the Sanbox environment. On the Sandbox there is no User-SSO - using rhsso instead. Also the namespace prefix is different. 
Intorduction of a new enviromental variable - `SANDBOX` and slight refactor of the scripts. 

## Verification
1. Install RHOAM by
```
INSTALLATION_TYPE=multitenant-managed-api make cluster/prepare/local
INSTALLATION_TYPE=multitenant-managed-api make code/run    
```
2. Once installation is complete ` watch -n5 "oc get rhmi rhoam -n sandbox-rhoam-operator -o yaml | yq e '.status.stage' -"` 
3. Install workload-web-app from this branch by:
```
export RHOAM=true 
export SANDBOX=true
export GRAFANA_DASHBOARD=true 
make local/deploy
```
4. Once console reports done, navigate to the grafana dashboards and check the `Workload App` dashboard - in ~2 minutes it must start showing data simmilar this the screenshot 
![image](https://user-images.githubusercontent.com/37799801/180776407-f7e72976-a9f4-4d74-aef4-233ab5155f50.png)
